pkg/tecdsa/signing/message.go

```diff
@@ -130,4 +130,24 @@ func (trfm *tssRoundFiveMessage) SenderID() group.MemberIndex {
 // marshaling purposes.
 func (trfm *tssRoundFiveMessage) Type() string {
 	return messageTypePrefix + "tss_round_five_message"
+}
+
+// tssRoundSixMessage is a message payload that carries the sender's
+// TSS round six components.
+type tssRoundSixMessage struct {
+	senderID group.MemberIndex
+
+	payload   []byte
+	sessionID string
+}
+
+// SenderID returns protocol-level identifier of the message sender.
+func (trsm *tssRoundSixMessage) SenderID() group.MemberIndex {
+	return trsm.senderID
+}
```

Comment:
I am not sure if this function is needed, please see [this](https://github.com/keep-network/keep-core/pull/3243#discussion_r964665190) comment.

Reply:
Same response as in https://github.com/keep-network/keep-core/pull/3243#discussion_r964665190